examples/angular/tsconfig.prod.json

```diff
@@ -5,7 +5,5 @@
     "declarationMap": false,
     "types": []
   },
-  "angularCompilerOptions": {
-    "enableIvy": false
-  }
```

Comment:
As mentioned in the changeset, customers need to delete this line after they migrate to Angular 12.